x-pack/plugins/reporting/public/components/report_info_button.tsx

```diff
@@ -145,6 +144,10 @@ export class ReportInfoButton extends Component<Props, State> {
           title: 'Status',
           description: get(info, 'status', NA),
         },
+        {
+          title: 'Browser Type',
+          description: get(info, 'browser_type', NA),
```

Comment:
It's not a huge deal, but it would be nice if this said `unknown` if the report in the listing is a PDF report that was created in an older version of Kibana where the `browser_type` isn't available. If the report is a CSV job, it should say `n/a`

Reply:
Ah, great point, forgot about our old friend CSV -- will adjust